src/pages/patch/patchTabs/CodeChanges.tsx

```diff
@@ -35,6 +36,19 @@ export const CodeChanges: React.FC = () => {
         const sortedFileDiffs = [...modCodeChange.fileDiffs].sort((a, b) =>
           a.fileName.localeCompare(b.fileName)
         );
+        const { fileDiffs } = modCodeChange;
+        const TotalFileDiff = {
```

Comment:
I think we should do away with the TotalFileDiff. The initial values are captured in the reduce function calls and it's nice if we can avoid mutating objects when we don't need to.

Reply:
Great idea.